crates/varcon/assets/README

```diff
@@ -59,7 +59,7 @@ is used when the spelling is almost generally considered a
 misspelling, and is only included for completeness.
 
 For Australian English "v" was used for variants that are widely used,
-but not preferred, and "V" for all "-or" (vs. "-our") variants and 
+but not preferred, and "V" for all "-or" (vs. "-our") variants and
```

Comment:
this file is vendored, please revert

Reply:
I guess `crates/wikipedia-dict/README.md` too then. Reverted